flang/include/flang/Runtime/descriptor.h

```diff
@@ -99,7 +99,7 @@ class DescriptorAddendum {
 public:
   explicit RT_API_ATTRS DescriptorAddendum(
       const typeInfo::DerivedType *dt = nullptr)
-      : derivedType_{dt} {}
+      : derivedType_{dt}, len_{0} {}
```

Comment:
Can this initializer be placed instead on the component declaration?  That's slightly better coding practice.